pkgs/tools/misc/nncp/default.nix

```
@@ -0,0 +1,46 @@
+{ stdenv, go, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "nncp";
+  version = "5.3.3";
+
+  buildInputs = [ go ];
+
+  preConfigure = ''
+    export GOCACHE=$PWD/.cache
+  '';
+
+  makeFlags = [
+    "PREFIX=$$out"
+    "CFGPATH=/etc/nncp.hjson"
+    "SENDMAIL=/run/wrappers/bin/sendmail"
+  ];
+
+  src = fetchurl {
+    url = "http://www.nncpgo.org/download/${pname}-${version}.tar.xz";
+    sha256 = "1l35ndzrvpfim29jn1p0bwmc8w892z44nsrdnay28k229r9dhz3h";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Secure UUCP-like store-and-forward exchanging";
+    longDescription = ''
+      This utilities are intended to help build up small size (dozens of
+      nodes) ad-hoc friend-to-friend (F2F) statically routed darknet
+      delay-tolerant networks for fire-and-forget secure reliable files,
+      file requests, Internet mail and commands transmission. All
+      packets are integrity checked, end-to-end encrypted, explicitly
+      authenticated by known participants public keys. Onion encryption
+      is applied to relayed packets. Each node acts both as a client and
+      server, can use push and poll behaviour model.
+
+      Out-of-box offline sneakernet/floppynet, dead drops, sequential
+      and append-only CD-ROM/tape storages, air-gapped computers
+      support. But online TCP daemon with full-duplex resumable data
+      transmission exists.
+    '';
+    homepage = http://www.nncpgo.org/;
```

```suggestion
    homepage = "http://www.nncpgo.org/";
```
Note: Funny they don't have a valid SSL certificate on their website, while they create an internet security related software :).

---

:) it seems they have a rather special setup.